Automattic/jetpack
# build: Post wpcom testing reminder on all Jetpack PRs

#### Changes proposed in this Pull Request:

When this was added in #25310, it was set up to only post on PRs that touched de-Fusioned files, with the intention that we'd update it once de-Fusioning was well under way. That time is now.

#### Other information:


#### Jetpack product discussion


p9dueE-5Me-p2#comment-8587 among others

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Make a PR based on this that does not build Jetpack. It should not have a comment added.
  * [x] https://github.com/Automattic/jetpack/actions/runs/3046842396/jobs/4910141281#step:7:21
* Update the PR to build Jetpack without touching any fusioned files. A comment should be added and updated.
  * [x] https://github.com/Automattic/jetpack/actions/runs/3046862259/jobs/4910188554#step:7:23
  * [x] https://github.com/Automattic/jetpack/actions/runs/3046862259/jobs/4910188554#step:13:27
* Update the PR again, still building Jetpack. The old comment should be detected, no new comment added.
  * [x] https://github.com/Automattic/jetpack/actions/runs/3046921749/jobs/4910323721#step:7:22
* Update the PR to no longer build Jetpack. The comment from the previous step should be deleted.
  * [x] https://github.com/Automattic/jetpack/actions/runs/3047007773/jobs/4910508140#step:7:22
